casepro/backend/junebug.py

```diff
@@ -115,7 +188,28 @@ def pull_contacts(
             tuple of the number of contacts created, updated, deleted and
             ignored
         """
-        return (0, 0, 0, 0)
+        identity_store = self.identity_store
+
+        # all identities created in the Identity Store in the time window
+        new_identities = identity_store.get_identities(
+            created_at__gte=modified_after, created_at__lte=modified_before)
```

Comment:
@imsickofmaps I'm inclined to make this a TODO or github issue to be tackled later. The search method is currently operational whereas the filters on the identity store need some work. Currently one can only filter on details but we need to be able to filter on a date range which requires a FilterSet to be implemented.